server/api/general.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const router = express.Router();
+import db from "../db";
+
+const passport = require("passport");
+
+
+
+router.get("/health", function (_, response) {
+	db.query("select version()")
+		.then((result) => {
+			if (result.rows.length > 0) {
```

Comment:
if rows is null this can cause an issue not sure if this scenario can happen, if yes you need to check if rows is not null or undefined